config/db.js

```diff
@@ -0,0 +1,20 @@
+const mongoose = require('mongoose')
+const config = require('config')
+const db = config.get('mongoURI')
```

Comment:
Perhaps a better name for this would be `dbConnectionString` or `mongoUri`?

Reply:
Renamed db to dbConnectionString.